MapboxNavigation/RouteMapViewController.swift

```diff
@@ -496,11 +496,7 @@ extension RouteMapViewController: MGLMapViewDelegate {
     }
 
     func updateShield(for controller: RouteManeuverViewController) {
-        let currentLegProgress = routeController.routeProgress.currentLegProgress
-
-        guard let upComingStep = currentLegProgress?.upComingStep else { return }
-        guard let ref = upComingStep.codes?.first else { return }
-        guard controller.shieldImage == nil else { return }
```

Comment:
@frederoni 👍 fixed in https://github.com/mapbox/mapbox-navigation-ios/pull/290/commits/a898ffa1abbefdddaf7f7259763b1d02a93f354f